apps/laboratory/tests/shared/fixtures/w3m-social-fixture.ts

```diff
@@ -3,8 +3,13 @@ import { ModalPage } from '../pages/ModalPage'
 import { ModalValidator } from '../validators/ModalValidator'
 import { timingFixture } from './timing-fixture'
 
-export const testMSocial = timingFixture.extend<ModalFixture>({
+export const testMSocial = timingFixture.extend<
+  ModalFixture & {
+    social: string[]
+  }
+>({
   library: ['wagmi', { option: true }],
+  social: ['github'],
```

Comment:
Added this with mind of keeping the list of available social tests then maybe we could iterate them one by one but it might not be the best place to do that. Yet, we have problem with other social providers since they have human verification and requires more investigation to tackle. 

I might revert this.

Reply:
Let's keep tests as flakyless as possible

Reply:
Exactly, in the `project.ts` I keep the social tests disabled for now